modules/base/include/modules/base/algorithm/image/layerramdistancetransform.h

```diff
@@ -27,8 +27,7 @@
  *
  *********************************************************************************/
 
-#ifndef IVW_LAYERRAMDISTANCETRANSFORM_H
-#define IVW_LAYERRAMDISTANCETRANSFORM_H
+#pragma once
 
 #include <modules/base/basemoduledefine.h>
 #include <inviwo/core/common/inviwo.h>
```

Comment:
You removed `inviwo.h` in most of the other files. Is it needed here?

Reply:
probably not, I'm slowly trying to remove the use of it. it just add some other includes, it is better to just include the needed headers directly